app/views/teaching_events/about_git_events.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= render Campaign::HeaderComponent.new(
+  title: ["Get Into Teaching", "events"],
+  colour: "yellow",
+  image: "media/images/content/hero-images/0025.jpg"
+) %>
+
+<div class="subject-specific">
+  <article>
+    <section class="statement">
+      <span class="pink xlarge bold">What to expect</span>
+    </section>
+
+    <section>
+      <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+
+      <p>They're a series of relaxed and informative events giving you the opportunity to get expert advice and talk to teachers and local training providers.</p>
+
+      <h2 class="heading-m">At the event</h2>
+
+      <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
```

Comment:
@GDW7 I wonder if we need to add more detail about 'presentations' (from the Events team) - this doesn't sound particularly exciting at the moment?

Also - I don't know whether these are scheduled at particular times - again, something we should maybe check for the individual event pages?

Reply:
@GDW7 I've checked with the Events team - the presentations are repeated, so saying 'drop in' is fine

The presentation topics are listed on the individual event detail pages so happy to leave this as is on the About page